### ISSUE TITLE
Host slideshows

### ISSUE THREAD

**Issue by maabernethy** [maabernethy/jake-academic-site#3]
Use Remark js and Mathjax for slideshows

**Comment by maabernethy** [comment]
@thejakeyboy I will need the photos that appear in your slideshows

**Comment by maabernethy** [comment]
@thejakeyboy Also, I am using a different CDN which is recommended on the [MathJax site](https://www.mathjax.org/cdn-shutting-down/) opposed to downloading it locally. If you still want it locally, I can do that too.

**Comment by maabernethy** [comment]
@thejakeyboy There is one block of mathjax that seems to not work - slide #11 in ec16 price discovery

**Comment by thejakeyboy** [comment]
OK I made a zip file of all of the talks I have posted on my previous web page: http://web.eecs.umich.edu/~jabernet/Talks/talk_directories.zip

It's a bit confusing, because in BDSI_2016 you'll notice a whole bunch of different versions of flint_water_talk.html, which are all versions of a Flint talk I've been giving everywhere. But all the image files etc. are in there.

In the future, it would be good to have an easy way to make a version of a talk, and perhaps still have it appear easily on my course page. Perhaps we should discuss this separately.
